spec/acceptance/suites/default/01_component_rpm_upgrade_spec.rb

```diff
@@ -94,12 +94,12 @@
         it 'should keep history of version changes in git log' do
           on(host, 'rm -rf /root/simp-beakertest*')
           on(host, "git clone #{mod_repo_url} simp-beakertest")
-          result = on(host, 'cd simp-beakertest; git log --pretty=format:"%an %ae %s"')
+          result = on(host, 'cd simp-beakertest; git log --pretty=format:"\'%an\' \'%ae\' \'%s\'"')
 
           expected = [
-            "simp_rpm_helper root@localhost.localdomain Imported version #{version1}",  # downgrade
-            "simp_rpm_helper root@localhost.localdomain Imported version #{version2}",  # upgrade
-            "simp_rpm_helper root@localhost.localdomain Imported version #{version1}",  # initial install
+            "'simp_rpm_helper' '' 'Imported version #{version1}'",  # downgrade
+            "'simp_rpm_helper' '' 'Imported version #{version2}'",  # upgrade
+            "'simp_rpm_helper' '' 'Imported version #{version1}'",  # initial install
```

Comment:
Why the extra pair of single-quotes in these lines? `simp_rpm_helper` is surrounded, as well as the `Imported version #{version1}` section, so in-between there are two spaces surrounding a pair: `''`